src/convert_handle.cpp

```diff
@@ -0,0 +1,59 @@
+#include "convert_handle.hpp"
+namespace vg {
+using namespace std;
+    void convert_handle_graph(const HandleGraph* converting, MutableHandleGraph* converted) {
+        assert(converted->node_size() == 0);
+        
+        // If xg is a null pointer, throw a runtime error
+        if (converting == nullptr) {
+            throw runtime_error("There is no graph to convert");
```

Comment:
You should probably also check whether `converted` is `nullptr` too, right? In fact, I also like to check whether the `converted` is an empty graph using `node_size()` and `get_path_count()`